app/src/pages/manage/frameworks/[frameworkId]/[competencyId]/index.vue

```diff
@@ -121,9 +121,12 @@ async function deleteCompetencyHandler(): Promise<void> {
     <div>Loading</div>
   </template>
   <template v-else-if="competency?.__typename == 'QueryCompetencySuccess'">
-    <UiBackbutton :to="`${parent.url}`">
-      {{ parent.title }}
-    </UiBackbutton>
+    <UiBreadcrumb>
+      <UiBreadcrumbLink to="/manage/frameworks">{{
+        t('frameworks.route.id.index.action.backButton')
+      }}</UiBreadcrumbLink>
```

Comment:
We can use the `v-t` attribute when it's a single child.

```suggestion
      <UiBreadcrumbLink
        v-t="'frameworks.route.id.index.action.backButton'"
        to="/manage/frameworks"
      />
```